packages/location-field/src/mocks/handlers.js

```diff
@@ -1,12 +1,19 @@
 import { rest } from "msw";
```

Comment:
Did you add this handler to `.storybook/preview.js`? I'm only getting an empty object back when I run the stories locally. I also don't get anything back at all for the reverse story.

https://github.com/opentripplanner/otp-ui/pull/328/files also adds a new msw endpoint

Reply:
Oh, I was testing the story by putting real API keys into the story controls. The use-case for the story, for me anyway, was to look at and compare the responses from HERE and Pelias for various searches, so mocking the responses didn't really make sense. At this stage, though, it might make more sense, so I'll look into adding that. You're probably getting weird responses because the API keys are not correct.  

I also haven't touched `preview.js` before, and I'm not sure what it does. I'll take a look and see if I can't figure that out!

Reply:
Nevermind my earlier comment. I didn't realize you were talking about location-field.

Reply:
A little confused here. The location-field is working fine for me, and the handler for it *is* making it into preview.js via the array that is passed in. I guessing that you are actually talking about the Geocoder package, in which case my original comment applies. I'll go ahead and create an MSW mock for it.

Reply:
OK, I believe I've taken care of what you're asking for in e4def13. Let me know if that works!

Reply:
This was a journey! But yes I was talking about creating mocks for the geocoder story and this looks excellent and works great thank you so much for implementing!